webapp/Connector/src/view/Learn.js

```diff
@@ -695,7 +702,8 @@ Ext.define('Connector.view.LearnHeader', {
     getDataView : function() {
         if (!this.headerDataView) {
             this.headerDataView = Ext.create('Connector.view.LearnHeaderDataView', {
-                cls: 'dim-selector',
+                flex: 6,
```

Comment:
Flex is specified for the dimension selector on the left side, but not on search field on the right, which will have no effect.